Linked_list/linked_list.test.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+const LinkedPackage = require('./linked_list.js');
+
+describe('Testing each method for Linked Lists', () => {
+  console.log(LinkedPackage.linkedList);
+  const LL = new LinkedPackage.linkedList();
+  LL.insert('bob');
+  LL.insert('Tim');
+  LL.insert('tom');
+  it('We should return a linked list when we run the functions', () => {
+    expect(LL).toEqual({
+      chainLength: 3,
```

Comment:
it is common to see this sort of test done over multiple lines, like so:
```
expect(LL.length).toBe(3));
expect(LL.head.value).toBe('tom');
expect(LL.head.next.value).toBe('bob');
```
etc, etc. This was a clever use of .toEqual plus an object literal though!